pkg/model/api/v1/user.go

```diff
@@ -20,10 +20,21 @@ import (
 	modelAPI "github.com/perses/perses/pkg/model/api"
 )
 
+type NativeProvider struct {
+	Password string `json:"password,omitempty" yaml:"password,omitempty"`
+}
+
+type OAuthProvider struct {
```

Comment:
No json, yaml?

Reply:
ah yes I will add them, even if the default works well, thanks

Reply:
updated